docs/Remote Control Guide/index.md

```diff
@@ -0,0 +1,185 @@
+
+View **ReleaseInteriorVehicleDataModule** in the [RPC Spec](https://github.com/smartdevicelink/rpc_spec#releaseinteriorvehicledatamodule)
+
+### SetGlobalProperties
+
+SetGlobalProperties is a request sent by a mobile app to inform SDL of a user's location within the vehicle. The request includes a `userLocation` parameter which contains a grid. The location of a user is important for SDL to know so it can determine whether or not a user is within a module's service area.
+
+View **SetGlobalProperties** in the [RPC Spec](https://github.com/smartdevicelink/rpc_spec#setglobalproperties) or the [HMI Documentation](https://smartdevicelink.com/en/guides/hmi/rc/setglobalproperties)
+
+## Remote Control Modules
+
+### Climate
+
+The climate module consists of climate sub-modules represented by a `ClimateControlCapabilities` object. Each sub-module exposes many aspects of a car's climate controls, such as setting the desired temperature or turning on the heated windshield. 
+
+### Radio
+
+The radio module consists of radio sub-modules represented by a `RadioControlCapabilities` object. Each sub-module exposes many aspects of a car's radio controls, such as setting the desired frequency and band the radio is operating on.
+
+### Seat
+
+The seat module consists of seat sub-modules represented by a `SeatControlCapabilities` object. Each sub-module exposes many aspects of a car's seat controls, such as setting the back tilt angle and the massage mode.
+
+### Audio
+
+The audio module consists of audio sub-modules represented by a `AudioControlCapabilities` object. Each sub-module exposes many aspects of a car's audio controls, such as setting the volume or modifying the equalizer settings.
+
+### Light
+
+The light module does not contain any sub-modules but instead has an array of `LightCapabilities` objects, each identified by a `LightName`. This module exposes the ability to modify attributes such as the brightness and color of each light.
+
+### HMI Settings
+
+The HMI settings module does not contain any sub-modules and is represented by an `HMISettingsControlCapabilities` object. This module exposes the the ability to set the desired temperature and distance units as well as toggle the display mode of the HMI between night and day.
+
+### Button
+
+Button is an interesting remote control component because it is not a remote control module. `RemoteControlCapabilities` includes an array of `ButtonCapabilities` structs which describe either a physical button or a softbutton. A mobile app may send a `ButtonPress` RPC with the `ButtonName` and `moduleId` from any of these `ButtonCapabilities` to perform an action on another remote control module.
+
+## Consent
+
+The behavior of module allocation in SDL Core is shown in the following table:
+
+!!! NOTE
+The driver is always considered to be within the service area.
+SDL will assume actions performed by the driver are consented to by the driver.
+!!!
+
```

Comment:
oh wow, interesting. can't argue with the core code but I don't see where that was changed in the evolution process since it was defined that background apps could control free modules in [0071](https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0071-remote-control-baseline.md)

Reply:
added in https://github.com/smartdevicelink/sdl_core_guides/pull/61/commits/4bfc036695cde776df4824670769a2273ec886b8